NorthwindCRUD/Models/Dtos/CategoryDto.cs

```diff
@@ -1,13 +1,18 @@
-﻿using NorthwindCRUD.Models.Contracts;
+﻿using System.ComponentModel.DataAnnotations;
+using NorthwindCRUD.Models.Contracts;
 
 namespace NorthwindCRUD.Models.Dtos
 {
     public class CategoryDto : ICategory
     {
+        [Range(1, int.MaxValue, ErrorMessage = "CategoryId must be greater than 0.")]
```

Comment:
CategoryId is autogenerated we should find a way to mark it as such or at least read only, but I believe the client shouldn't enter any value for it.